### PR TITLE
feat(Table): update paths to reference deprecated directory

### LINKED ISSUE #330: Table - update path to deprecated directory
Follow up to breaking change issue https://github.com/patternfly/patternfly-react/issues/8005

Write a script to update the path to import from deprecated barrel file.

_Required actions:_
1. Build codemod
2. Build test
3. Update readme with description & example

### PR DESCRIPTION
**Pull request by jenny-s51**
Closes #330

### CODE REVIEW

File: packages/pf-codemods/README.md [hunk 1]
```diff
@@ -1552,6 +1552,17 @@ Out:
...
+
+We've deprecated the current implementation of Table. In order to continue using this deprecated implementation, the import path must be updated to our deprecated package and specifiers must be aliased. However, we suggest updating to our composable Table implementation.
+
+The following imports will be affected by this deprecation:
+
+- Table
+- TableBody
+- TableHeader
+- TableProps
+
```
Comment (thatblindgeye) [inline packages/pf-codemods/README.md:1565]: Can we also include examples for the input/output?

**Review by thatblindgeye — COMMENTED** [review]
I think this looks good, just a change noted below. The main thing is that this and the codemod that updates TableComposable to just Table will cause this codemod to run after the rename (if a consumer is running fixes more than once).

Wondering if we could update the logic to moveSpecifiers a little so that if other composable Table components are found in the import declaration to skip the import path rename. Since I can't imagine a consumer *only* importing Table without Th, Td, etc, which those wouldn't be imported from the new deprecated Table.

If we go that route we'd need to decide whether we'd want to include another argument to be passed in (a condition or array of components that tell the helper "if true or if these components are in the import declaration, don't rename the import path", or just add table logic internally.

**Review by jenny-s51 — COMMENTED** [review]
Great points @thatblindgeye , thank you - I've updated #388 to add the `data-codemods="true"` attribute to `Table` which is the method that we were using in v4 to prevent multiple updates to codemods, thanks @gitdallas for this suggestion

File: packages/pf-codemods/README.md [hunk 2]
```diff
@@ -1552,6 +1552,35 @@ Out:
...
+Out:
+
+```jsx
+import {
+ Table as TableDeprecated,
+ TableBody as TableBodyDeprecated,
+ TableHeader as TableHeaderDeprecated,
+ TableProps as TablePropsDeprecated
+} from '@patternfly/react-table/deprecated';```
```
Comment (wise-king-sullyman) [inline packages/pf-codemods/README.md:1582]: I think if the backticks aren't on a newline they'll cause some formatting issues.

**Review by wise-king-sullyman — COMMENTED** [review]
One markdown formatting nit:

**Review by wise-king-sullyman — APPROVED** [review]

**Review by gitdallas — APPROVED** [review]

**Review by thatblindgeye — APPROVED** [review]
Just needs a rebase

**Review by jenny-s51 — COMMENTED** [review]
@thatblindgeye rebased - should be all set to merge

**Review by wise-king-sullyman — APPROVED** [review]

**Pull request merged**